### DISCUSSION TITLE
No suitable completed repeats found

### DISCUSSION THREAD

**Discussion by francovanvuuren** [NotePlan/plugins#317] [Q&A]
Good day
When completing a task, I run the /rpt command. However, I get the message "No suitable completed repeats found"

The task looks as follows
[x] Patrol Shifts @repeat(1m) #buurtwag @done(2022-06-30)

I am running Noteplan Version 3.5.2 (793) on a MacOS

**Comment by jgclark** [comment]
Hi. As the README says, the /generate repeats command only works on lines with @repeat(...) and @done(YYYY-MM-DD HH:MM). Your line doesn't have the HH:MM part.  The command removes the HH:MM as the only possible way in NotePlan of keeping track of which completed repeats have been re-generated.  I hope that helps.
